### DISCUSSION TITLE
Allow uploading custom background images

### DISCUSSION THREAD

**Discussion by ghost** [JulianPrieber/littlelink-custom#283] [Feature requests]
Currently it seems that in order to set a custom background image you have to create a whole new theme. It'd be nice to instead have the ability to upload a custom background image to use with the current (or indeed any) theme. I'm imagining a setting for the background image with the following options:

- Theme default (whatever background the theme specifies)
- Colour (solid or gradient (like button colour options))
- Upload image

**Comment by JulianPrieber** [comment]
Custom background images are definitely coming, if everything goes well we will have this feature in the next feature-update.
